src/routing/http-routing-v1.md

```diff
@@ -199,15 +217,38 @@ Access-Control-Allow-Origin: *
 Access-Control-Allow-Methods: GET, OPTIONS
 ```
 
+## Known Schemas
+
+This section contains a non-exhaustive list of known schemas that MAY be supported by clients and servers.
+
+### Peer
+
+The `peer` schema represents an arbitrary peer.
+
+
+```json
+{
+  "Schema": "peer",
+  "ID": "12D3K...",
+  "Addrs": ["/ip4/..."],
+  "Protocols": ["transport-bitswap", ...]
+}
+```
+
+- `ID`: the [Peer ID](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md).
+- `Addrs`: a list of known [multiaddrs][multiaddr] for this peer. This list MAY be incomplete.
+- `Protocols`: an OPTIONAL field with the transports known to be used by this peer. If empty, it means the server does not know and the client can use the `identify` protocol to learn more.
```

Comment:
@hacdias  Unsure if this field should be allowed to be `undefined`. There is no precedence for this in this API, all other fields are always present. Would it be safer to always return an empty list here?

```suggestion
- `Protocols`: a list of protocols known to be supported by this peer. 
  - If empty, it means the server is missing protocol information, and the client should use `ID` and `Addrs` to connect to the peer and use the [libp2p identify protocol](https://github.com/libp2p/specs/tree/master/identify) to learn about supported ones.
```

Reply:
That's also okay.